source/guides/clear/mixer.rst

```diff
@@ -345,61 +345,100 @@ Refresh your web server to see the update content for mix version 20.
 You can also look in ~/mixer/update/www/<mix version> to see the update
 content in your workspace.
 
-
 Example 4: Build an image
 =========================
 
 This example shows how to build a bootable image containing the
 :command:`kernel-kvm`, :command:`os-core`, and the :command:`os-core-update`
-bundles from Example 2. Underneath, mixer uses `ister`_ to generate the
+bundles from Example 2. Underneath, mixer uses `clr-installer`_ to generate the
 image.
 
 #. Change directory into your mix.
 
 #. Configure image.
 
-   Edit the ister configuration file for your image to include all of the bundles you want pre-installed in the image. If this is the first time creating an image, first get a copy of the
-   :file:`release-image-config.json` template file:
+   Create a YAML configuration file to specify aspects of your image
+   such as image name, target media, bundles, etc.  See `Installer YAML Syntax`_ 
+   for more information on clr-installer configuration YAML syntax.
+
+   For this example, we will download a sample YAML and modify it.
 
    .. code-block:: bash
 
-      curl -O https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json
+      curl -O https://raw.githubusercontent.com/clearlinux/clr-installer/master/scripts/kvm.yaml
```

Comment:
Hi @bktan8, can you please review this error? I'm not sure of the root cause. 
After following Example 4, stdout shows:

`Writing partition table to: loop0 [success]
Updating partition table for: loop0 [success]
Adjusting filesystem configurations [success]
Writing vfat file system to loop0p1 '/boot' [success]
Writing swap file system to loop0p2 [success]
Writing ext4 file system to loop0p3 '/' [success]
Writing mount files [success]
Installing base OS and configured bundles [*failed*]
ERROR: Installation has failed!****
`

Reply:
`Error Trace:
errors.Wrap()
     errors/errors.go:91
swupd.(*SoftwareUpdater).OSInstall()
     swupd/swupd.go:362
2020/02/04 10:36:35 ERROR: Couldn't build image: exit status 1`

Reply:
Ignore. This was a result of a missing step.